Classes/ViewHelpers/Translate/LabelsViewHelper.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace SMS\FluidComponents\ViewHelpers\Translate;
+
+use TYPO3\CMS\Extbase\Utility\LocalizationUtility;
+use TYPO3\CMS\Fluid\Core\ViewHelper\Exception\InvalidVariableException;
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+use TYPO3Fluid\Fluid\Core\ViewHelper\Traits\CompileWithRenderStatic;
+
+class LabelsViewHelper extends AbstractViewHelper
```

Comment:
Is there an actual need for this? Looks like a convenience shortcut to just translate a bunch of labels in one call.

Reply:
Yup, it's just a nice helper to make the fluid templates more readable.